[DOCS] Preview rules privilege update (#1588)
elastic/security-docs: docs/detections/rules-ui-create.asciidoc

[[preview-rules]]
=== Preview your rule (optional)

> You can preview custom query, threshold, machine learning, or event correlation rules to get feedback on how noisy the rule will be before submitting it. You can then fine-tune the query, if needed, to reduce the number of alerts that may come in.

Reviewer: ```suggestion
You can preview custom query, threshold, machine learning, or event correlation rules to learn how noisy a rule will be before submitting it. If necessary, you can then adjust the rule's query.
```
